_sass/_typography.scss

```diff
@@ -0,0 +1,7 @@
+@import url(https://fonts.googleapis.com/css?family=Lato:400,700);
+@import url(https://fonts.googleapis.com/css?family=Inconsolata);
```

Comment:
@tysongach Was it you that said there's some performance benefit of using a `<link>` tag instead of `@import`ing? Have a link?